packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -460,6 +460,16 @@ export async function activate(context: vscode.ExtensionContext) {
 
     // Api
     const internalApi: any = {
+      channelService,
```

Comment:
Missing methods needed by the Apex extension, Apex Replay Debugger and Apex Debugger.